### PR TITLE
PT-1104 loop mode legend

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: css/tideline.less [hunk 1]
```diff
@@ -430,12 +422,29 @@ svg {
...
     }
   }
+
   // only in legends
-  &.d3-rect-basal-undelivered {
-    fill: none;
-    stroke: @basal;
-    stroke-dasharray: 1.5, 3;
-    stroke-width: 1.5;
+  &.d3-basal-loop-mode {
```
Comment (franck-fourel) [inline css/tideline.less:431]: replacing d3-basal-automated with d3-basal-loop-mode is breaking the display of the automated basal
image

**Review by franck-fourel — CHANGES_REQUESTED** [review]
display of automated basal is broken since you removed the css class that is used in js/plot/basal.js
```
return invisible ? 'd3-basal d3-basal-invisible' : `d3-basal d3-rect-basal d3-basal-${d.deliveryType}`;
```

File: css/tideline.less (hunk 1 above)
Comment (yvon-dblg) [reply css/tideline.less:431]: I don't see what is broken. Can you provide a screenshot with difference (same as above but without the problem) ?

**Review by franck-fourel — APPROVED** [review]
LGTM

**Pull request merged**